README.md

```diff
@@ -40,34 +40,28 @@ You can install this program globally with `npm i -g zksync-cli` or run the comm
 
 ### ⚙️ Options (flags)
 
-- `--zeek`: zeek, the dev cat, will search for an inspirational quote and provide to you at the end of any command.
 - `--l1-rpc-url`: override the default L1 rpc URL when `localnet` is selected as the network. Usage `--l1-rpc-url=http://...`.
 - `--l2-rpc-url`: override the default L2 rpc URL when `localnet` is selected as the network. Usage `--l1-rpc-url=http://...`.
+- `--zeek`: zeek, the dev cat, will search for an inspirational quote and provide to you at the end of any command.
 
 ## 👩‍💻 Developing new features
 
 ### Install and build
 
-1. Install all dendencies with `npm i`.
+1. Install all dependencies with `npm i`.
 2. This project was build with Typescript. Run `npm run build` to compile code in `/src` into `/bin`.
-
-To create a version run:
-
-```sh
-git tag v0.1.3    // with the correspondent tag
-git push --tags  
-```
+3. You can run your local build with `node ./bin`
```

Comment:
Please post a screenshot/evidence of this working with parameters like `--l1-rpc-url` and `--l2-rpc-url` in the PR description